packages/ui/src/lobbyPlayerBanner/lobbyPlayerBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Title } from "../player/title";
+import { LobbyPlayer } from "@creature-chess/models";
+
+
+type Props = {
+	player: LobbyPlayer | null;
+};
+
+const LobbyPlayerBanner: React.FunctionComponent<Props> = ({ player }) => {
+	if (!player) {
+		return (
+			<div key={Math.floor(Math.random() * 1000)} className="player bot">
+				<span className="name-and-image">
+					<img
+						src={"https://creaturechess.jamesmonger.com/images/ui/no_player_img.png"}
+						alt="no player image"
+					/>
+					<div className="spacer" />
+					empty slot
+				</span>
+			</div>
+		);
+	}
+	return (
+		<div key={player?.id} className="player">
```

Comment:
Move the key up a level, individual components should not know about their own key, the key is the concern of the thing rendering multiple of them